app/Models/Manufacturer.php

```diff
@@ -23,8 +23,8 @@ class Manufacturer extends SnipeModel
     protected $rules = [
         'name'   => 'required|min:2|max:255|unique:manufacturers,name,NULL,id,deleted_at,NULL',
         'url'   => 'url|nullable',
-        'support_url'   => 'url|nullable',
         'support_email'   => 'email|nullable',
+        'support_url'   => ['regex:/.+:\/\/.+/','nullable'],
```

Comment:
As I mentioned on the dev call, we *might* be able to simplify this a bit, using the `starts_with` validation.

`'support_url' => 'nullable|starts_with:http://,https://',afp://,facetime://,file://,irc://`